map_model/src/map.rs

```diff
@@ -1063,4 +1065,136 @@ impl Map {
         }
         result
     }
+
+    // TODO This returns a mixture of different things related to the turn. It might be better
+    // to create and return a `Turn`.
```

Comment:
Hmm, yeah, maybe even a new struct would be OK here. Still reading through, not sure if I understand yet if the full `Turn` makes sense or not